### PR TITLE
Add DSC_event_scraper

### LINKED ISSUE #452: Script to Scrap Community Website
# Description

Create a Script to scrap the Community Website for [Developer Students Club](https://dsc.community.dev/). The DSC Community Website lists various Events that are happening across the globe. Scrap them and save them as a CSV File.

## Language

- [X] Python
- [X] Javascript
- [X] Golang

## Type of issue

_Please delete options that are not relevant._

- [X] Feature

### PR DESCRIPTION
**Pull request by DevChoganwala**
# Description

This scripts scrapes all the events available on the DSC community website and outputs them in a file called data.csv (located in the same directory as your script).
Modules Used: `requests`, `csv`, `sys`, `json`

Fixes #452 

## Type of change

- [X] New feature (non-breaking change which adds functionality)

# Checklist:

- [X] My code follows the style guidelines(Clean Code) of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by DevChoganwala** [comment]
@HarshCasper Please let me know if I have pulled requests incorrectly since I am new to contributing. I will make appropriate changes. Thank you.

File: Python/DSC_event_scraper/event_scraper.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            sys.exit()
+        out = response.json()
+        for i in out['results']:
+            li = [i['title'],i['chapter']['title'],i['chapter']['country_name'],i['chapter']['city'],i['chapter']['timezone'],i['start_date_naive'][:10],i['start_date_naive'][11:],i['end_date_naive'][:10],i['end_date_naive'][11:],i['url'],i['event_type_title']]
+            writer.writerow(li)
+        url = out['links']['next']
+        if(url == None): #url = None when there is no further page
+            break
+
+
```
Comment (HarshCasper) [inline Python/DSC_event_scraper/event_scraper.py:25]: Remove these extra lines

File: Python/DSC_event_scraper/event_scraper.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        try:
+            response.raise_for_status() #if exceptions raised quit the program
+        except:
+            sys.exit()
+        out = response.json()
+        for i in out['results']:
+            li = [i['title'],i['chapter']['title'],i['chapter']['country_name'],i['chapter']['city'],i['chapter']['timezone'],i['start_date_naive'][:10],i['start_date_naive'][11:],i['end_date_naive'][:10],i['end_date_naive'][11:],i['url'],i['event_type_title']]
+            writer.writerow(li)
+        url = out['links']['next']
+        if(url == None): #url = None when there is no further page
```
Comment (HarshCasper) [inline Python/DSC_event_scraper/event_scraper.py:22]: Keep the Comment above the Line of Code

File: Python/DSC_event_scraper/event_scraper.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import requests
+import csv
+import sys
+import json
```
Comment (HarshCasper) [inline Python/DSC_event_scraper/event_scraper.py:4]: I don't think that the imports are properly stacked up. Can you try running your Script through PyLint and review what changes you can do here?

File: Python/DSC_event_scraper/event_scraper.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+url = 'https://dsc.community.dev/api/event/?fields=id,chapter,event_type,event_type_title,title,status,start_date,end_date,start_date_naive,end_date_naive,url&status=Published&page=1'
+with open('data.csv','w') as csvfile:
+    writer = csv.writer(csvfile)
+    header = ['title','chapter name','chapter country','chapter city','timezone','start date','start time(local)','end date','end time(local)','event url','event type']
+    writer.writerow(header) 
+    while(1):
+        response = requests.get(url)
+        try:
+            response.raise_for_status() #if exceptions raised quit the program
```
Comment (HarshCasper) [inline Python/DSC_event_scraper/event_scraper.py:14]: Kindly keep the Single Line Comment above the line of code

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by DevChoganwala** [comment]
@HarshCasper I have made the above commented changes to my script. Please check if I need to make more changes before merging. Thanks you.

**Comment by HarshCasper** [comment]
give @DevChoganwala 75 points

**Pull request merged**